examples/dns/zone.yaml

```diff
@@ -1,16 +1,13 @@
 apiVersion: dns.yandex-cloud.jet.crossplane.io/v1alpha1
 kind: Zone
 metadata:
-  name: example-zone
+  name: yc-courses-1999
 spec:
+  deletionPolicy: Orphan
   forProvider:
-    name: example-zone
-    description: desc
-    labels:
-      label1: "label-1-value"
-    zone: "example.com."
-    public: false
-    privateNetworksRefs:
-      - name: example-net
+    folderId: <your-folder-id>
+    name: yc-courses-1999 # укажи имя своей dns-зоны
+    public: true
+    zone: pushka.ga. # укажи свою dns-зону
```

Comment:
I beleve that not all users will be allowed to create `pushka.ga` zone.
And `example.com` was made for testing purpose - https://www.iana.org/domains/reserved

Reply:
Fixed. I get it.